projects/driver_controls/src/gpio_expander.c

```diff
@@ -49,6 +58,7 @@ StatusCode gpio_expander_init(GpioExpanderStorage *expander, I2CPort port, GpioE
   memset(expander, 0, sizeof(*expander));
   expander->port = port;
   expander->addr = MCP23008_ADDRESS + addr;
+  expander->int_pin = *interrupt_pin;
```

Comment:
Nit: `NULL` checks? Although, we haven't really been consistent about this since most of the time people don't try `NULL`s in most fields. We should probably decide on this for the next round?

Reply:
True. We should probably add NULL checks to everything.